packages/shorebird_cli/test/src/command_runner_test.dart

```diff
@@ -67,13 +68,21 @@ void main() {
 
     group('--version', () {
       test('outputs current version and engine revisions', () async {
+        final versionResult = await Process.run('flutter', ['--version']);
```

Comment:
I don't think we want to spawn a process from our test – it would make this test a fair bit slower and makes this test depend on a correctly configured environment (which I believe is why it's failing in CI). We use `ShorebirdProcess` as a wrapper for `Process` as a way to get around this elsewhere in tests. See the `testProcess` parameter we supply to our Commands in other tests, such as `shorebird/packages/shorebird_cli/test/src/commands/release_command_test.dart`.

Reply:
Ok, I'll fix that. thx for the review

Reply:
> I don't think we want to spawn a process from our test – it would make this test a fair bit slower and makes this test depend on a correctly configured environment (which I believe is why it's failing in CI). We use `ShorebirdProcess` as a wrapper for `Process` as a way to get around this elsewhere in tests. See the `testProcess` parameter we supply to our Commands in other tests, such as `shorebird/packages/shorebird_cli/test/src/commands/release_command_test.dart`.

I a bit confused on how it should be used on my test here, because here I am actually just testing an option of the 
`ShorebirdCliCommandRunner` rather than a `ShorebirdCommand` which in that case, I understand that I can pass it the shorebird process but in the case of a `CompletionCommandRunner` I can't manage to find a solution. 
It would be great if can give me a bit more indication on they way to implement this. 

Thank you, in advance